Dockerfile

```diff
@@ -0,0 +1,43 @@
+# Dockerfile to build and test DeepDive inside a container
+#
+# `make build-in-container test-in-container` uses master image built by this.
+# `util/build/docker/` contains utilities relevant to this.
+FROM ubuntu
```

Comment:
There are pros and cons.  I think the source code should be portable, even for non-ubuntu, so I'd vote against fixing it.  Any compelling reason to do so?